### DISCUSSION TITLE
vscode cannot authorize

### DISCUSSION THREAD

**Discussion by tanhh326** [community/community#18840] [Copilot Conversations] [answered]
version : v1.30.6165

Recently, I changed to a new computer and synchronized the plug-in, but when I log in to the plug-in authorization, it prompts such a message. Why is this? And it doesn't open the browser prompting me for authorization
![d5d5c2b3233ce3e8ed0d1aa413eb2af](https://user-images.githubusercontent.com/82007768/174427886-69e1f7bf-3195-4878-a028-f86e3f1d5f69.png)

**Comment by davecheney** [comment, marked as answer]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Comment by alamin-karno** [comment]
You need to ensure your user profile has access to connected apps like Salesforce CLI.

**Comment by KYJKY** [comment]
It is the process of linking VScode and github.
If you click that, it will open github site and it asks if you want to allow that linking.
But, you can't open it, I suggest several methods.

1. Temporarily replace another browser.
2. Open vscode with administrator privileges.

**Comment by tanhh326** [comment]
Okay, I can board, thank you for your answer.  (๑′ᴗ‵๑)
